tools/READMEs/templates/README.md.j2

```diff
@@ -0,0 +1,53 @@
+# {{manifest.name}} for YunoHost
+
+[![Integration level](https://dash.yunohost.org/integration/{{manifest.id}}.svg)](https://dash.yunohost.org/appci/app/{{manifest.id}}) ![](https://ci-apps.yunohost.org/ci/badges/{{manifest.id}}.status.svg)  ![](https://ci-apps.yunohost.org/ci/badges/{{manifest.id}}.maintain.svg)
+[![Install {{manifest.id}} with YunoHost](https://install-app.yunohost.org/install-with-yunohost.svg)](https://install-app.yunohost.org/?app={{manifest.id}})
+
+*[Lire ce readme en français.](./README_fr.md)*
+
+> *This package allows you to install {{manifest.id}} quickly and simply on a YunoHost server.
+If you don't have YunoHost, please consult [the guide](https://yunohost.org/#/install) to learn how to install it.*
+
+## Overview
+
+{{manifest.description[lang]}}
+
+**Shipped version:** {{manifest.version}}
+
+{% if upstream.demo %}**Demo:** {{upstream.demo}}{% endif %}
+
+{% if screenshots %}
+## Screenshots
+
+{% for screenshot in screenshots %}
+   ![](./doc/screenshots/{{screenshot}})
+{% endfor %}
+{% endif %}
+
+{% if disclaimer %}
+## Disclaimers / important information
+
+{{ disclaimer }}
+{% endif %}
+
+## Documentation and resources
+
+{% if upstream.website %}* Official app website: {{ upstream.website }}{% endif %}
+{% if upstream.userdoc %}* Official user documentation: {{ upstream.userdoc }}{% endif %}
+{% if upstream.admindoc %}* Official admin documentation: {{ upstream.admindoc }}{% endif %}
+{% if upstream.code %}* Upstream app code repository:  {{ upstream.code }}{% endif %}
+* YunoHost documentation for this app: https://yunohost.org/app_{{manifest.id}}
+* Report a bug: https://github.com/YunoHost-Apps/{{manifest.id}}_ynh/issues
+
+## Developer info
+
+Please send your pull request to the [testing branch](https://github.com/YunoHost-Apps/{{manifest.id}}_ynh/tree/testing).
```

Comment:
Add blocks to all sections. So it's possible to overwrite everything

Reply:
Mokay but there's not really any design for this ... the current design doesn't allow the app to override the structure of the README, it's really about fetching info from manifest.json, and injecting the contents from doc/screenshots/ and doc/DISCLAIMER.md